spec/features/user_visits_dashboard_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rspec_helper'
+
+feature "user visits dashboard" do
+  scenario "it can see its favorite users" do
+    user_1, user_2, user_3 = create_list(:user, 3)
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user_1)
+
+    co_1, co_2 = create_list(:company, 2)
+  end
```

Comment:
Did this test not get finished?

Reply:
There is a bug here. I'll PR again tomorrow. I'll close it for now.